hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/functional/TestCleanPlanExecutor.java

```diff
@@ -99,6 +99,206 @@ private static Stream<Arguments> argumentsForTestKeepLatestCommits() {
     );
   }
 
+  private static Stream<Arguments> argumentsForTestTriggerCleanEveryNCommits() {
+    return Stream.of(
+            Arguments.of(1),
+                Arguments.of(2),
+                Arguments.of(3),
+                Arguments.of(4)
+    );
+  }
+
+  @ParameterizedTest
+  @MethodSource("argumentsForTestTriggerCleanEveryNCommits")
```

Comment:
nit: make the name consistent?

Reply:
so this method is intended to supply the values for the config `hoodie.clean.max.commits` with the documentation `Number of commits after the last clean operation, before scheduling of a new clean is attempted.` Hence I kept this name. @yihua 

I am open to hearing your thoughts.